collect_app/src/main/java/org/odk/collect/android/views/HierarchyElementView.java

```diff
@@ -16,83 +16,55 @@
 
 import android.content.Context;
 import android.graphics.drawable.Drawable;
-import android.view.Gravity;
+import android.view.View;
 import android.widget.ImageView;
 import android.widget.RelativeLayout;
 import android.widget.TextView;
 
+import org.odk.collect.android.R;
 import org.odk.collect.android.logic.HierarchyElement;
 import org.odk.collect.android.utilities.TextUtils;
-import org.odk.collect.android.widgets.QuestionWidget;
 
 public class HierarchyElementView extends RelativeLayout {
 
     private TextView primaryTextView;
     private TextView secondaryTextView;
-    private ImageView icon;
+    private ImageView imageView;
 
+    public HierarchyElementView(Context context) {
+        super(context);
+    }
 
     public HierarchyElementView(Context context, HierarchyElement it) {
```

Comment:
The second parameter of this constructor could probably have a more meaningful name. I know it's kinda obvious that it's a HierarchyElement but if I see it somewhere else I might forget :sweat_smile: